environments/exercise9.r

```diff
@@ -0,0 +1,33 @@
+### Write an enhanced version of str() that provides more information about functions. Show where the function was found and what environment it was defined in.
+
+myStr <- function(name) {
+  list(
+    str=capture.output(str(name)),
```

Comment:
my PEP8 reflexes made me do this.
Is there a good R linter that adheres to our style guide?